TESTING.md

```diff
@@ -0,0 +1,25 @@
+# Testing Elm-mdl
+
+### This is only relevant when developers and contributors to Elm-mdl
+
+Tests are located in a separate *"tests"* folder and does not affect elm-mdl.
+
+Prerequisite: To be able to run tests from your terminal - install node-test-runner by running:
+> npm install -g elm-test
+
+Run all tests by executing this command in the root folder:
+>elm-test
+
+The travis.yml file has been configured to run elm-test when any branch is pushed to GitHub
+
+**NOTE**
+
+One primary reason to include elm-test now, is to improve the speed of Travis build.
+Currently, there are no actual tests implemented. The Material components and Demo components are imported by the testrunner and only verifies that building those modules does not fail.
+
+We should look at other projects to see how to implement tests in Elm-mdl.
```

Comment:
I can do that, to confirm - should I create an issue that we need testing, and then add the information from the TESTING.md into the issue?